### PR TITLE
Add automatic inventory generator

### PR DESCRIPTION
**Pull request by ivanovmi**
Based on kargo inventory generator.
Adding support for:
 * glusterfs nodes
 * elasticsearch nodes
 * keepalived nodes

Added "--force" option (only for development)

### CODE REVIEW

File: generator.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+    parser.add_argument(
+        '--nodes', dest='nodes_list', metavar='N', nargs='+',
+        required=True, help='List of nodes'
+    )
+    parser.add_argument(
+        '--elastics', dest='elastics_list', metavar='N', nargs='+', default=[],
+        help='Number of elastic search nodes'
+    )
+    parser.add_argument(
+        '--keepaliveds', dest='keepaliveds_list', metavar='N',
```
Comment (corest) [inline generator.py]: replace 'keepaliveds' with 'registries'

**Comment by corest** [comment]
Also, i would move this script to the utils folder. Who knows, maybe we will add some kind of cli-tool wrapper around playbooks for better usability. 
So, it will be like this:

```
automation:
 \
  utils
  \ 
    inventory-generator
  inventory
  \ 
    inventory.cfg
  roles
```

**Comment by ivanovmi** [comment]
@corest fixed, please review

**Comment by corest** [comment]
Add default inventory location - inventory/inventory.cfg

**Comment by corest** [comment]
Here is output from the command:
```
utils/inventory-generator -i inventory/inventory.cfg --nodes kargo1 kargo2 kargo3
 [ERROR]: You should set at least 2 masters

 [ERROR]: You should set at least 3 nodes for etcd clustering

 [ERROR]: You should set at least 3 nodes for elasticsearch

 [ERROR]: You should set at least 3 nodes for glusterfs cluster

 [ERROR]: You should set at least 2 nodes for keepalived
```

It will be better to automatically fill required groups with added nodes by default. For example, with 3 nodes above, generator (without explicitly specified parameters) should create groups like this:
```
[keepalived]
kargo1
kargo2
kargo3

[glusterfs]
kargo1
kargo2
kargo3

[elasticsearch]
kargo1
kargo2
kargo3

[etcd]
kargo1
kargo2
kargo3

[elasticsearch]
kargo1
kargo2
kargo3
```

So, if no other options specified - take first minimum required nodes from group 'all' and assign them to required group (e.g. - first three for etcd, firsh two for keepalived)

**Comment by corest** [comment]
Merged into https://github.com/seecloud/automation/pull/11/commits/eed45df22c674b7e9abfffc815f9612590ee25a6

**Pull request closed without merging**